### PR TITLE
Add headers field to request object (in selenium-webdriver) (fixing #29077)

### PR DESCRIPTION
**Pull request by danielthegray**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.) *(I don't see any tests for the request sending)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/http/index.js>> send(httpRequest) on line 101 clearly has a `headers` attribute which is not defined in the type here.
- [ ] Increase the version number in the header if appropriate. *(it's not for another version of Selenium)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. *(No substantial changes)*

### CODE REVIEW

**Comment by cnishina** [comment]
Looks okay. The PR references the httplib.Request which is this file:
https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/lib/http.js#L79

Should we also add the other params while we are at it?
```
method: string;
path: string;
data: Object;
```

**Review by cnishina — CHANGES_REQUESTED** [review]
Let's add the other params here. See comment.

File: types/selenium-webdriver/http.d.ts [hunk 1]
```diff
@@ -21,6 +21,8 @@ export class Request {
      */
     constructor(method: string, path: string, opt_data?: Object);
 
+    headers: Map<string, string>;
+
```
Comment (cnishina) [inline types/selenium-webdriver/http.d.ts]: method: string
path: string
data: Object

Comment (thedanielgray) [reply types/selenium-webdriver/http.d.ts]: I will add these fields below the headers definition.

**Comment by thedanielgray** [comment]
> Looks okay. The PR references the httplib.Request which is this file:
> https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/lib/http.js#L79

Thanks a lot for pointing out the original constructor! I was wondering where the actual constructor definition was and was having a hard time finding it. Cheers!

**Comment by thedanielgray** [comment]
I've realized that the Response object is also missing some field definitions (namely status, body, and headers). Should I add them here and rename the PR or rather create a separate PR?

**Comment by cnishina** [comment]
This is fine. Create a separate PR and I'll review that one. Thanks!

**Review by cnishina — APPROVED** [review]

**Pull request merged**